config/mfCommonDefs.mk

```diff
@@ -52,8 +61,8 @@ PACKAGE_VER_MINOR ?= $($(ShortPackageLoc)_VER_MINOR)
 PACKAGE_VER_PATCH ?= $($(ShortPackageLoc)_VER_PATCH)
 
 #BUILD_VERSION ?= 1
-BUILD_VERSION:= $(shell $(BUILD_HOME)/$(Project)/config/build/tag2rel.sh | awk '{split($$0,a," "); print a[4];}' | awk '{split($$0,b,":"); print b[2];}')
-PREREL_VERSION:= $(shell $(BUILD_HOME)/$(Project)/config/build/tag2rel.sh | awk '{split($$0,a," "); print a[8];}' | awk '{split($$0,b,":"); print b[2];}' )
+BUILD_VERSION:= $(shell $(BUILD_HOME)/config/build/tag2rel.sh | awk '{split($$0,a," "); print a[4];}' | awk '{split($$0,b,":"); print b[2];}')
```

Comment:
Does this suggests a redefinition of `$BUILD_HOME` or changing the way of `config` submodule insertion to the main repository? Also, I see now the `config` being a part of the main repo - shouldn't it be in the submodule?

Reply:
Usage of `BUILD_HOME` in upstream macros changed (now points to the equivalent of the `cmsgemos` directory, rather than the directory in which `cmsgemos` lived), and I've long wanted to get rid of any external (outside of `make`) dependency on a bunch of environment variables.
Now, there *should* be less setup to get things to compile (could probably be cleaned up further in the future)